src/css/helpers.js

```diff
@@ -0,0 +1,38 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+// expands a plotcss selector
+exports.buildFullSelector = function buildFullSelector(selector) {
```

Comment:
On second thought, would you mind moving this file and `src/css/plotcss_injector.js` to `src/lib/`. I would prefer leaving the `src/css/` directory for css (and scss) files only. 

I would also vote for merging the two files into one and naming it something like `plotcss_utils.js`

Sorry for the confusion.